pkgs/applications/misc/dfilemanager/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, qt5, file, kde5}:
+
+let
+  version = "git-2015-06-10";
+in
+stdenv.mkDerivation rec {
+  name = "dfilemanager-${version}";
+  src = fetchgit {
+    url = "git://git.code.sf.net/p/dfilemanager/code";
+    rev = "806a28aa8fed30941a2fd6784c7c9c240bca30e3";
+    sha256 = "1k15qzjmqg9ffv4cl809b071dpyckf8jspkhfhpbmfd9wasr0m7i";
+  };
+
+  buildInputs = [ cmake qt5.base qt5.tools qt5.x11extras file kde5.solid];
+
+  cmakeFlags = "-DQT5BUILD=true";
+
+  meta = {
+    homepage = "http://dfilemanager.sourceforge.net/";
```

Comment:
I know. The contrib guide seems undecided on this and uses a mixed style. Can you point me at something that makes some sort of argument for/against this? Some sort of style guide maybe?